playbooks/tasks/portal-role-task-mongo-check-docker-service.yml

```diff
@@ -83,6 +83,7 @@
       eval: "db.serverStatus()"
       until: "'\"ok\" : 1' in mongo_shell_result.transformed_output"
       retries: 20
+      ignore_errors: True
   when: mongo_service_ok
   register: mongo_checking_admin_result
```

Comment:
Since we are ignoring errors here, shouldn't we be adding an `is defined` check for where this variable is being used to avoid undefined errors?

Reply:
I link line 102

Reply:
`ignore_errors: True` in this case and below
do not belong to `inlcude_task` task like this:

```yaml
include_tasks: tasks/portal-role-task-mongo-shell-eval.yml
ignore_errors: True
register: mongo_checking_admin_result
```

They are passed down (to a task in `tasks/portal-role-task-mongo-shell-eval.yml`) via `mongodb_shell.ignore_errors` to

```yaml
community.mongodb.mongodb_shell:
register: mongo_shell_result
ignore_errors: "{{ mongodb_shell.ignore_errors | default(False) }}"
```

The result which is affected is not `mongo_checking_admin_result` or `checking_replicaset_result`
but `mongo_shell_result` and it is handled with:

`"{{ not (mongo_shell_result.failed | default(False)) }}"` in both cases.